internal/core/timers.rs

```diff
@@ -143,6 +143,23 @@ impl Timer {
             .map(|timer_id| CURRENT_TIMERS.with(|timers| timers.borrow().timers[timer_id].running))
             .unwrap_or(false)
     }
+
+    /// Change the duration of timer. If the timer was previously started by calling [`Self::start()`]
+    /// with a duration and callback, then the time when the callback will be next invoked
+    /// is re-calculated to be in the specified duration relative to when this function is called.
+    ///
+    /// Does nothing if the timer was never started.
+    ///
+    /// Arguments:
+    /// * `interval`: The duration from now until when the timer should fire. And the period of that timer
+    ///    for [`Repeated`](TimerMode::Repeated) timers.
```

Comment:
We don't usually use this style to document the arguments.

Reply:
I recall trying this style once before, but at least I wasn't consistent about it. I quite like it, TBH. I don't recall where I saw it first.

Reply:
I just copy the comment from Timer::start() 😆